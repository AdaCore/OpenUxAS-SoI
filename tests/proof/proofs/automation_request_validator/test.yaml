filenames: ["uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads"]