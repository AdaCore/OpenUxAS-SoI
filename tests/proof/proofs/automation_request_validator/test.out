uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:15:06: info: data dependencies proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:17:16: info: postcondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:24:06: info: data dependencies proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:26:16: info: postcondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:33:06: info: data dependencies proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:35:16: info: postcondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:35:17: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:42:06: info: data dependencies proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:45:08: info: postcondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:47:55: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:49:55: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:62:10: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:62:51: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:64:13: info: loop invariant initialization proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:64:13: info: loop invariant preservation proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:65:49: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:65:112: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:110:13: info: loop invariant initialization proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:110:13: info: loop invariant preservation proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:115:16: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:116:13: info: initialization of "Request_Q_Tmp" proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:122:55: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:137:19: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:138:22: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:140:23: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:141:22: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:143:16: medium: precondition might fail, cannot prove Length (Container) < Container.Capacity
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:143:47: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:144:31: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:145:35: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:146:36: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:147:36: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:149:19: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:150:22: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:150:63: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:151:22: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:152:31: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:154:16: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:158:26: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:158:26: medium: assertion might fail, cannot prove Prove_Same_Requests_Valid (This.Configs, M.Get (Model (Request_Q_Tmp), I + 1).Content, M.Get (Model (This.Requests_Waiting_For_Tasks), I).Content) [possible explanation: loop invariant at line 109 should mention Request_Q_Tmp]
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:160:26: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:160:57: info: overflow check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:161:26: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:171:10: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:190:13: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:191:13: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:192:13: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:193:13: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:195:16: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:220:14: info: initialization of "V" proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:235:07: info: initialization of "UniqueAutomationReq" proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:236:07: info: initialization of "Id" proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:241:22: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:246:22: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:255:35: info: tag check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:256:35: info: tag check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:259:48: info: tag check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:260:39: medium: precondition might fail, cannot prove Length (Container) < Container.Capacity [possible explanation: precondition of subprogram at uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:305 should mention This]
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:264:26: info: tag check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:266:25: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:268:13: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:268:13: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:277:35: info: tag check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:280:48: info: tag check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:284:26: info: tag check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:287:26: info: tag check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:288:39: medium: precondition might fail, cannot prove Length (Container) < Container.Capacity [possible explanation: precondition of subprogram at uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:305 should mention This]
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:291:25: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:293:13: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:293:13: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:300:13: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:303:39: medium: precondition might fail, cannot prove Length (Container) < Container.Capacity [possible explanation: precondition of subprogram at uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:305 should mention This]
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:307:25: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:309:13: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:309:13: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:313:10: info: initialization of "IsReady" proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:314:10: info: initialization of "Request_Q_Tmp" proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:319:25: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:322:13: medium: precondition might fail, cannot prove Length (Container) < Container.Capacity
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:324:28: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:326:16: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:328:23: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:330:26: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:331:26: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:333:16: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:333:16: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:334:17: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:395:36: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:396:09: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:396:09: info: discriminant check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:397:23: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:398:12: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:398:51: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:417:45: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:420:25: medium: precondition might fail, cannot prove Tail'Length <= To.Capacity - Length (To) [possible explanation: loop invariant at line 426 should mention ReasonForFailure]
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:421:47: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:422:35: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:422:54: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:427:25: info: loop invariant preservation proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:427:25: info: loop invariant initialization proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:429:88: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:429:108: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:434:13: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:435:29: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:444:19: info: loop invariant initialization proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:444:19: info: loop invariant preservation proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:445:72: info: overflow check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:446:85: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:446:105: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:447:68: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:447:88: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:451:52: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:466:67: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:469:49: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:475:31: info: loop invariant preservation proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:475:31: info: loop invariant initialization proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:476:35: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:476:62: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:481:22: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:482:76: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:483:72: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:487:22: medium: precondition might fail, cannot prove Tail'Length <= To.Capacity - Length (To) [possible explanation: loop invariant at line 443 should mention ReasonForFailure]
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:489:30: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:489:49: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:494:16: medium: assertion might fail, cannot prove IsReady = Check_For_Required_Entity_Configurations (Entity_Ids => EntityIds, Configurations => This.Configs.Available_Configuration_Entity_Ids, States => This.Configs.Available_State_Entity_Ids, Planning_States => Get_PlanningStates_Ids (Request))
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:502:13: medium: precondition might fail, cannot prove Tail'Length <= To.Capacity - Length (To)
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:503:29: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:506:16: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:507:71: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:509:16: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:516:13: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:523:25: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:537:20: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:539:22: info: loop invariant initialization proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:539:22: info: loop invariant preservation proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:539:40: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:540:80: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:541:42: info: loop invariant initialization proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:541:42: info: loop invariant preservation proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:542:23: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:543:24: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:543:89: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:548:25: info: loop invariant initialization proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:548:25: info: loop invariant preservation proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:548:43: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:549:27: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:549:93: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:550:45: info: loop invariant preservation proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:550:45: info: loop invariant initialization proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:557:19: medium: precondition might fail, cannot prove Tail'Length <= To.Capacity - Length (To)
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:558:35: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:565:19: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:566:35: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:568:19: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:569:35: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:574:13: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:582:10: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:599:31: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:603:21: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:610:56: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:613:28: medium: precondition might fail, cannot prove Tail'Length <= To.Capacity - Length (To) [possible explanation: loop invariant at line 618 should mention ReasonForFailure]
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:615:36: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:615:63: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:619:28: info: loop invariant preservation proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:619:28: info: loop invariant initialization proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:622:86: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:622:108: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:627:57: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:630:28: medium: precondition might fail, cannot prove Tail'Length <= To.Capacity - Length (To) [possible explanation: loop invariant at line 635 should mention ReasonForFailure]
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:632:36: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:632:64: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:636:28: info: loop invariant preservation proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:636:28: info: loop invariant initialization proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:639:87: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:639:110: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:644:19: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:657:16: medium: precondition might fail, cannot prove Tail'Length <= To.Capacity - Length (To)
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:659:24: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:660:24: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:663:19: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:673:10: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:692:16: info: loop invariant initialization proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:692:16: info: loop invariant preservation proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:694:59: info: overflow check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:696:57: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:696:75: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:701:83: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:702:65: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:702:83: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:705:43: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:712:24: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:717:34: info: discriminant check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:719:51: info: discriminant check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:721:31: medium: precondition might fail, cannot prove Tail'Length <= To.Capacity - Length (To) [possible explanation: loop invariant at line 691 should mention ReasonForFailure]
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:723:39: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:723:60: info: discriminant check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:724:39: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:729:34: info: discriminant check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:731:51: info: discriminant check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:733:31: medium: precondition might fail, cannot prove Tail'Length <= To.Capacity - Length (To) [possible explanation: loop invariant at line 691 should mention ReasonForFailure]
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:735:39: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:735:60: info: discriminant check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:736:39: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:741:34: info: discriminant check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:743:51: info: discriminant check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:745:31: medium: precondition might fail, cannot prove Tail'Length <= To.Capacity - Length (To) [possible explanation: loop invariant at line 691 should mention ReasonForFailure]
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:748:39: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:748:60: info: discriminant check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:749:39: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:755:25: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:763:34: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:765:19: medium: precondition might fail, cannot prove Tail'Length <= To.Capacity - Length (To) [possible explanation: loop invariant at line 691 should mention ReasonForFailure]
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:766:57: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:767:27: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:771:19: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:773:52: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:778:78: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:779:60: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:783:22: info: assertion proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:787:13: info: initialization of "ErrResponseID" proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:795:13: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:796:13: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:797:13: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:798:13: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:799:13: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.adb:801:39: medium: precondition might fail, cannot prove Contains (Container, Key) [possible explanation: postcondition of call at line 794 should mention ErrResponseID (for argument ErrResponseID)]
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:29:45: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:29:58: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:52:35: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:52:57: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:60:49: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:60:71: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:67:16: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:82:38: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:86:38: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:99:19: info: discriminant check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:100:74: info: discriminant check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:102:19: info: discriminant check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:103:74: info: discriminant check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:105:19: info: discriminant check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:106:75: info: discriminant check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:109:06: info: data dependencies proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:122:38: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:122:57: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:127:31: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:128:46: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:128:65: info: range check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:130:06: info: data dependencies proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:156:06: info: data dependencies proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:162:72: info: tag check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:164:73: info: tag check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:165:48: info: tag check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:171:14: info: postcondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:176:77: info: tag check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:178:78: info: tag check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:179:53: info: tag check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:187:70: info: tag check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:189:71: info: tag check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:190:46: info: tag check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:200:60: info: tag check proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:211:11: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:213:11: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:215:11: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:217:11: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:227:28: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:230:28: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:233:32: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:243:45: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:250:09: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:252:18: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:262:60: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:263:60: info: precondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:273:08: info: postcondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:280:07: info: initialization of "IsReady" proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:286:08: info: postcondition proved
uxas-comms-lmcp_net_client-service-automation_request_validation-spark.ads:311:10: info: postcondition proved
